components/Activity.js

```diff
@@ -2,7 +2,7 @@ import Image from 'next/image'
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
 import { faCalendarAlt, faClock, faMapMarkedAlt, faUser, faUsers } from '@fortawesome/free-solid-svg-icons'
 
-export default function Atividade(props) {
+export default function Ativity(props) {
```

Comment:
I think you should destructe the props into the values you want to use.

Like { data, ...}

Reply:
Done. And thanks for the tip 👍